src/Nethermind/Nethermind.JsonRpc/ErrorType.cs

```diff
@@ -91,5 +91,10 @@ public static class ErrorCodes
         /// Unsupported fork error
         /// </summary>
         public const int UnsupportedFork = -38005;
+
+        /// <summary>
+        /// Invalid attributes
+        /// </summary>
+        public const int InvalidAttributes = -38003;
```

Comment:
There is `MergeErrorCodes`, I think this and `UnsupportedFork` should be moved there